.github/workflows/merge.yml

```diff
@@ -0,0 +1,66 @@
+#
+# DeGirum GitHub CI build script
+# Copyright 2023 DeGirum Corporation
+#
+# PySDKExamples merge workflow: runs tests on PySDKExamples install scripts and jupyter notebooks
+#
+
+name: CI Merge
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  build:
+    # Setup matrix on Ubuntu, Windows and MacOS runners
```

Comment:
Misleading comment: MacOS is not in matrix

Reply:
removed macos from comment and ifs